packages/commonwealth/client/scripts/views/components/component_kit/CWBannerText/CWBannerText.scss

```diff
@@ -0,0 +1,17 @@
+.linkTextRow {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.link {
```

Comment:
we can't do this, because those classes are not encapsulated in this case and then classes clash on each other. We usually use parent css class that wraps the component (in this case "RenderTextWithLink"). Then we nest any other class inside.

.RenderTextWithLink {
  .linkTextRow {}

   etc...
}

Reply:
i used the scss practices in RenderTextWithLink.scss